app/src/main/java/com/goldenraven/padawanwallet/wallet/WalletActivity.kt

```diff
@@ -74,7 +116,11 @@ class WalletActivity : AppCompatActivity() {
         val intentSeedPhrase: Intent = Intent(this, DrawerActivity::class.java).putExtra("drawerItem", "seedphrase")
         val intentSendCoinsBack: Intent = Intent(this, DrawerActivity::class.java).putExtra("drawerItem", "sendCoinsBack")
 
-
+        if (offerFaucetCallDone == false) {
+            tour = TourGuide.init(this).with(TourGuide.Technique.Click)
+                    .setPointer(Pointer().setGravity(Gravity.LEFT))
+                    .playOn(binding.toolbar)
```

Comment:
Like for the drawer button I pointed the tour to the toolbar button and aligned it's gravity to left.